docs/dvl/dvl-troubleshooting.md

```diff
@@ -0,0 +1,60 @@
+# DVL Troubleshooting
+
+## Is my DVL faulty?
+
+Below is a small list to check if your DVL is faulty or you are experiencing any issues. It starts with the basic before moving on to more advanced troubleshooting. When you come to the point of failure or the troubleshooting do not help, please submit a support ticket to our support team through our [Support portal](https://waterlinked.com/support). Please write what is the issue and how you have tested the DVL. Please describe how the test environment is setup and how the DVL is mounted. Please add any [Diagnostic reports](gui/diagnostic-report.md) and pictures of the test setup. This will help our support team greatly! 
+
+If the trouble shooting guide do not help, please check out [FAQ DVL](faq.md#faq-dvl)!  
+
+### LED
+1. Put the DVL under water and power it on with an adequate power supply.
```

Comment:
Maybe put the suggestion for power supply as a separate point to emphasize it?

Reply:
[4cc23a5](https://github.com/waterlinked/waterlinked.github.io/pull/30/commits/4cc23a58715602dcbfbcc5c73f60f5808ebf526f)